package.json

```diff
@@ -70,7 +70,8 @@
     "@babel/plugin-transform-runtime": "^7.10.3",
     "@babel/preset-env": "^7.10.3",
     "@babel/runtime-corejs3": "^7.10.3",
-    "@okta/okta-auth-js": "4.6.2",
+    "@okta/okta-auth-js": "4.7.0",
```

Comment:
4.7.1?

Reply:
Any reason we don't use flexible minors here?